engine/graphics/src/vulkan/graphics_vulkan.cpp

```diff
@@ -903,7 +904,7 @@ namespace dmGraphics
 
             selected_device = device;
             selected_queue_family = queue_family;
-            break;
+            break; // Why do we break here? x_x
```

Comment:
I guess because we're happy with the device we've found?
Are we not?

Reply:
Ah, lol I'm a dummy. Yes we are, I thought we didn't loop through all devices but I was probably ~~drunk~~ tired when I read the code